scripts/rook.sh

```diff
@@ -53,11 +53,17 @@ function deploy_rook() {
 		cat "${TEMP_DIR}"/cluster-test.yaml
 		kubectl_retry create -f "${TEMP_DIR}/cluster-test.yaml"
 	fi
-	rm -rf "${TEMP_DIR}"
 
 	kubectl_retry create -f "${ROOK_URL}/toolbox.yaml"
 	kubectl_retry create -f "${ROOK_URL}/filesystem-test.yaml"
 	kubectl_retry create -f "${ROOK_URL}/pool-test.yaml"
+	
+	# TODO: to be removed once issue is closed - https://github.com/rook/rook/issues/13040
+	# Create the default `csi` subvolumegroup
+	curl -o "${TEMP_DIR}/subvolumegroup.yaml" "${ROOK_URL}/subvolumegroup.yaml"
+	sed -i "s|name:.*|name: csi|g" "${TEMP_DIR}/subvolumegroup.yaml"
```

Comment:
You are missing the `kubectl_retry create -f` call to create the group and remove this subvolumegroup in teardown.

Reply:
Oops, missed it😅